docs/contributing/local-development/Local_Linking.mdx

```diff
@@ -45,6 +46,25 @@ Now, you should be able to run ESLint in the local downstream repository as you
 To check that the local package is being used, consider adding a `console.log("Hello, world!");` to a file such as `./packages/eslint-plugin/dist/index.js` and making sure that log appears when linting the local downstream repository.
 :::
 
+## Testing Rules
+
+Now that you've linked the `@typescript-eslint/*` packages with your local downstream repository, the next step would be to include the rule on the local repository ESLint configuration file, e.g:
+
+```json title=".eslintrc.json"
+{
+  "rules": {
+    "@typescript-eslint/your-awesome-rule": "error"
+  }
+  // ...
+}
+```
+
+After that, you need to run your repository's `lint` script and your changes should be reflected on the project.
+
+:::caution
```

Comment:
At first I tried to use a note but I got the feeling that it was not visible enough, I agree that caution is a little intense here maybe the tip callout could be a great fit? Since we are giving some tips on the plugin build, what do you think?

Reply:
I'd be fine with a note or a tip yeah. There's not much info on this page so I don't think we have to worry about folks only reading a bit of the docs and then stopping before it.